src/components/run-button-container.js

```diff
@@ -387,8 +312,11 @@ export function RunButtonContainer({ studyUuid, currentNode, disabled }) {
                     startComputationAsync(
                         ComputingType.VOLTAGE_INIT,
                         null,
-                        () => startVoltageInit(studyUuid, currentNode?.id),
-                        () => setRanVoltageInit(true),
+                        () => {
+                            setComputationStopped(false);
```

Comment:
call directly startVoltageInit because `setComputationStopped` call has been moved in `startComputationAsync` directly